src/etc/inc/interfaces.inc

```diff
@@ -3117,11 +3117,14 @@ function interface_dhcpv6_prepare($interface = 'wan', $wancfg)
     $rtsoldscript .= "\techo \${2} > /tmp/{$wanif}_defaultgwv6\n";
     $rtsoldscript .= "fi\n";
     $rtsoldscript .= "if [ -f /var/run/dhcp6c_{$wanif}.pid ]; then\n";
-    $rtsoldscript .= "\t/bin/pkill -F /var/run/dhcp6c_{$wanif}.pid\n";
+    $rtsoldscript .= "\tdhcp6c_pid=\$(cat \"/var/run/dhcp6c_{$wanif}.pid\")\n";
+    $rtsoldscript .= "\t/bin/kill -HUP \${dhcp6c_pid}\n";
```

Comment:
FWIW, there is such a thing:

    /bin/pkill -HUP -F /var/run/dhcp6c_{$wanif}.pid